src/state/GlobalState.tsx

```diff
@@ -142,9 +142,9 @@ export class GlobalState extends PureComponent<Props> {
             ? t('notify.install.canceled')
             : t('notify.install.finished')
         notify([title, message])
-        return this.refresh()
+        return this.refreshLibrary()
```

Comment:
Why was this changed?

Reply:
Because refresh doesnt refresh the files.

Reply:
This is highly counterintuitive. But yeah I got it.